activiti-core/activiti-bpmn-converter/src/test/resources/checkConversionPrefix.bpmn

```diff
@@ -4,7 +4,10 @@
         <bpmn2:startEvent id="StartEvent_1">
             <bpmn2:outgoing>SequenceFlow_0afdugi</bpmn2:outgoing>
         </bpmn2:startEvent>
-        <bpmn2:manualTask id="Task_0cshed6" name="Task"></bpmn2:manualTask>
+        <bpmn2:manualTask id="Task_0cshed6" name="Task">
+        	<bpmn2:incoming>SequenceFlow_0afdugi</bpmn2:incoming>
```

Comment:
The indentation contains tab characters. User spaces only.

Reply:
fixed